finder-app/Makefile

```diff
@@ -0,0 +1,18 @@
+default: writer_compile
+	gcc -Wall -Werror writer.o -o writer
+	
+writer_compile: clean
+	gcc -c writer.c
+	
+clean:
+	rm -f writer.o writer
+	
+CROSS_COMPILE: make_cross_compile
+	aarch64-none-linux-gnu-gcc -Wall -Werror writer.o -o writer -L/home/pranav/install-lnx/gcc-arm-10.2-2020.11-x86_64-aarch64-none-linux-gnu/bin
```

Comment:
@prvbharadwaj1731 see notes from lecture content regarding specifying CROSS_COMPILE for compatibility with future assignments.
Also you won't be able to use an absolute linker path, this will break when attempting to use outside your build host.  I wouldn't expect this to be needed.